Add page for Zarr datasets (#86)
zarr-developers/zarr-developers.github.io: datasets/index.md

---
layout: single
author_profile: false
title: Zarr Datasets
sidebar:
  title: "Content"
  nav: sidebar
---

<font size="4">

> According to the <a href="https://zarr.dev/adopters/">website</a>, Zarr is

Reviewer: This reads a bit odd. I would just mention "adopters" directly.

utilised by a variety of research groups, universities, organisations, and
open-source communities.<br><br>

> We have compiled a list of public datasets for the community to use.<br><br>

Reviewer: I'd avoid "we" here since we're also asking for submissions. Maybe something about this being a sampling of datasets, i.e. not complete or anything.

> We have compiled a list of public datasets for the community to use.<br><br>

Reviewer 2: ```suggestion
Below is a list of a fews examples of public datasets.<br><br>
```

</font>

| Dataset                                                                                                           | Hosting               | Size  |
| ----------------------------------------------------------------------------------------------------------------- | --------------------- | ----- |
| [CMIP6](https://console.cloud.google.com/marketplace/details/noaa-public/cmip6)                                   | Google Cloud          | 1 PB  |

> | [IDR (OME-NGFF)](https://idr.github.io/ome-ngff-samples/)                                                         | EBI                   | 3 TB  |

Reviewer: Maybe just "Image Data Resource (IDR)"

| [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/catalog?filter=zarr)                       | Microsoft Azure       |       |
| [MUR Sea Surface Temperature](https://registry.opendata.aws/mur)                                                  | AWS Open Data Program | 54 TB |
| [NASA POWER](https://registry.opendata.aws/nasa-power/)                                                           | AWS Open Data Program |       |
| [National Water Model](https://discourse.pangeo.io/t/the-national-water-model-reanalysis-zarr-dataset-on-aws/1449)| Amazon AWS            | 5 TB  |

> | [NGFF](https://ngff.openmicroscopy.org/data/index.html)                                                           | Various providers     |       |

Reviewer: Maybe this could be in another section as "other catalogs".

Author: Thanks for the good suggestion! 

How about we keep [NGFF](https://ngff.openmicroscopy.org/data/index.html) where it is now and separate it out into 'Catalogs' sections once we have a few more examples?

Reviewer: Then I'd rather just remove it for the moment. We can always add it later.

| [webKnossos](https://zarr.webknossos.org/)                                                                        | scalableminds GmbH    | 70 TB |

<font size="4">

> <br>Feel free to add new datasets by sending a PR to the website <a href="https://github.com/zarr-developers/zarr-developers.github.io/">repository</a>.<br><br>

Reviewer 2: ```suggestion
<br>If you think your dataset would be a useful example to add to the list, feel to send a PR to the website <a href="https://github.com/zarr-developers/zarr-developers.github.io/">repository</a>.<br><br>
```

---
layout: single
author_profile: false
title: Zarr Datasets
sidebar:
  title: "Content"
  nav: sidebar
---

<font size="4">

> Zarr is <a href="https://zarr.dev/adopters/">adopted</a> by a variety of

Reviewer: ```suggestion
Zarr has been <a href="https://zarr.dev/adopters/">adopted</a> by a variety of
```

research groups, universities, organisations, and open-source communities.<br>

> 
> Below is a list of a few examples of public datasets.<br><br>

Reviewer: ```suggestion
Below is a list of a few examples of public datasets.<br><br>
```

| Dataset                                                                                                           | Hosting               | Size  |
| ----------------------------------------------------------------------------------------------------------------- | --------------------- | ----- |
| [CMIP6](https://console.cloud.google.com/marketplace/details/noaa-public/cmip6)                                   | Google Cloud          | 1 PB  |
| [Image Data Resource (IDR)](https://idr.github.io/ome-ngff-samples/)                                              | EBI                   | 3 TB  |
| [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/catalog?filter=zarr)                       | Microsoft Azure       |       |
| [MUR Sea Surface Temperature](https://registry.opendata.aws/mur)                                                  | AWS Open Data Program | 54 TB |
| [NASA POWER](https://registry.opendata.aws/nasa-power/)                                                           | AWS Open Data Program |       |
| [National Water Model](https://discourse.pangeo.io/t/the-national-water-model-reanalysis-zarr-dataset-on-aws/1449)| Amazon AWS            | 5 TB  |

> | [NGFF](https://ngff.openmicroscopy.org/data/index.html)                                                           | Various providers     |       |

Reviewer: ```suggestion
```

| [webKnossos](https://zarr.webknossos.org/)                                                                        | scalableminds GmbH    | 70 TB |

<font size="4">
<br>If you think your dataset would be a useful example to add to the list,
feel free to send a PR to the website <a href="https://github.com/zarr-developers/zarr-developers.github.io/">repository</a>.<br><br>

Please ensure the dataset you're adding to the list is public, free to use,

> documented, represents Zarr's use case in a specific domain, and is stable/maintained. 

Reviewer: Not sure I would understand "represents Zarr's use case in a specific domain. I would also expand on "is stable/maintained", e.g. "Please only include URLs that you expect to be stable and maintained for ..." (the next several years?)

Author: Thanks for the feedback!
Made some changes. Check here: https://zarr-developersgithubio--86.org.readthedocs.build/en/86/datasets/
